TheFactorial.py

```diff
@@ -0,0 +1,9 @@
+def factorial(n):
+    if n == 0 or n == 1:
+        return 1
+    result = n * factorial(n - 1)
+
+    return result
+
+
+print("The factorial of 5 is:  " + str(factorial(5)))
```

Comment:
Instead of  factorial(5) use a generic value that is read from keyboard.

Reply:
Done